R/shinyui.R

```diff
@@ -25,6 +25,13 @@ withMathJax <- function(...) {
 }
 
 renderPage <- function(ui, showcase=0, testMode=FALSE) {
+  # Check if ui has lang attribute; otherwise, NULL
+  lang <- if ("lang" %in% names(attributes(ui))) {
```

Comment:
Simpler to do:

```R
lang <- attr(ui, "lang", exact = TRUE)
```

(This will default to `NULL` if there's no such attribute)